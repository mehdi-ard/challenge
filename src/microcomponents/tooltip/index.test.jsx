import React from 'react';
import renderer from 'react-test-renderer';
import Tooltip from './';
import Button from 'microcomponents/button';

it('renders default', () => {
  expect(
    renderer
      .create(
        <Tooltip title="Tooltip">
          <Button>Tooltip</Button>
        </Tooltip>
      )
      .toJSON()
  ).toMatchSnapshot();
});
it('renders placement', () => {
  expect(
    renderer
      .create(
        <Tooltip placement="top" title="Tooltip">
          <Button>Tooltip</Button>
        </Tooltip>
      )
      .toJSON()
  ).toMatchSnapshot();
});
