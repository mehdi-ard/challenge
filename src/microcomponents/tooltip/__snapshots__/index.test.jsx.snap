// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default 1`] = `
<button
  className="ant-btn"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  type="button"
>
  <span>
    Tooltip
  </span>
</button>
`;

exports[`renders placement 1`] = `
<button
  className="ant-btn"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  type="button"
>
  <span>
    Tooltip
  </span>
</button>
`;
